### PR TITLE
OBPIH-3141: Implement Beta of Stock Movement Redesign for Outbound

### PR DESCRIPTION
**Pull request by bamoure**

### CODE REVIEW

**Comment by awalkowiak** [comment]
@bamoure is this ticket ready to be reviewed and potentially ready to be merged? The snapshot tests for wizards steps failed (you need to regenerate jest snapshot for it and it will be passing)

**Comment by bamoure** [comment]
@awalkowiak I updated the tests, the PR should be ready for review. I'm waiting on Kelsey's approval before it is ready to be merged.

File: src/js/components/stock-movement-wizard/StockMovement.jsx
Comment (awalkowiak) [inline src/js/components/stock-movement-wizard/StockMovement.jsx:130]: Why the name was removed?

File: src/js/components/stock-movement-wizard/StockMovement.jsx
Comment (awalkowiak) [inline src/js/components/stock-movement-wizard/StockMovement.jsx:137]: Why you want to have it both on pack and send page?

File: src/js/components/stock-movement-wizard/StockMovement.scss
Comment (awalkowiak) [inline src/js/components/stock-movement-wizard/StockMovement.scss:247]: No new line at the end of file ;)

**Review by awalkowiak — COMMENTED** [review]
@bamoure, @jmiranda codewise it looks good. There are only two-three minor issues/questions I wrote here and I also added small suggestions in the comment in the Jira ticket.

File: src/js/components/stock-movement-wizard/StockMovement.jsx
Comment (bamoure) [reply src/js/components/stock-movement-wizard/StockMovement.jsx:130]: This was asked for in the design

File: src/js/components/stock-movement-wizard/StockMovement.jsx
Comment (bamoure) [reply src/js/components/stock-movement-wizard/StockMovement.jsx:137]: This is so the label will appear on the send page in all cases: page 6 when it has packing support and page 5 otherwise

**Review by awalkowiak — APPROVED** [review]

**Pull request merged**
